### PR TITLE
Clean Back-office templates, part 5 - orders

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | See below.
| Type?             | improvement
| Category?         | BO
| BC breaks?        | This PR should be considered as a BC because of lot of HTML structure changes and they might be used by a module or anything else, but it's acceptable. The optimizations are only within the files itself, so it does not matter if anyone uses the old version in any form in their modules.
| Deprecations?     | no
| How to test?      | Go through Orders, Customers and Addresses sections in BO and see if nothing is broken by this PR. There should be no visible change.
| Possible impacts? | No bad impacts. :-)

### Description
- I went through all templates and removed 99% of unneeded boostrap code like `<div class="row"><div class="col">XXX</div></div>`. It will reduce render depth and complexity by several layers - easier to navigate, debug CSS, maybe faster to render on slower PCs.
- I fixed all places where somebody used single quotations or no quotations at all.
- I fixed all places where somebody combined row and column in same element, this is not allowed.
- I fixed all places where somebody used syntax line col-sm-12 col-md-12 or col-md-12 col-lg-6. The first one is not needed.
- Small fixes here and there - removed some container classes, fixed module position page responsivity a bit, fixed virtual product block responsivity etc...
It should produce no BC breaks, unless some 3rd party module was targetting some classes they shouldn't have OR they targeted it by child or parent.

### Templates concerned
Sell/Address
Sell/Customer
Sell/Customer/View
Sell/CustomerService/CustomerThread
Sell/CustomerService/MerchandiseReturn
Sell/CustomerService/OrderMessage
Sell/Order/Cart
Sell/Order/Cart/View
Sell/Order/CreditSlip
Sell/Order/Delivery
Sell/Order/Invoices
Sell/Order/Order
Sell/Order/Order/Create
Sell/Order/Order/View

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26064)

### CODE REVIEW

**Review by NeOMakinG — DISMISSED** [review]
@Julievrz no wording changes

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/view.html.twig
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/view.html.twig:51]: For this one I have doubts since it's only populated via a hook will it no break the display if modules expected to add a list of columns?

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig]: Extra line

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig]: Extra line

**Review by jolelievre — COMMENTED** [review]
Thanks @Hlavtox 
I had a few questions though before approving this one

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/view.html.twig
Comment (Hlavtox) [reply src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/view.html.twig:51]: @jolelievre You are right, if modules use boostrap col, they will have 15px extra padding on both sides. It should be considered as a small BC, but for the good purpose. I think the modules should not care about what is around the hook they hook into.

If we keep the .row and I just want to display a paragraph with my module, it will move it by -15px to both sides.

Comment (jolelievre) [reply src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/view.html.twig:51]: Ok for me, then you just need to rebase (there is a conflict) And maybe remove the extra lines and it's ok for me!

**Comment by Hlavtox** [comment]
@jolelievre Lines cleared, rebased. 👍

**Review by jolelievre — DISMISSED** [review]
Thanks @Hlavtox

File: admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.ts
Comment (NeOMakinG) [inline admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.ts:136]: A bit worried about this, did you test it? Not sure that the d-none class is added at the right place now

Comment (Hlavtox) [reply admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.ts:136]: @NeOMakinG If I didn't mess it up during rebase, it should work fine. :-)

**Review by NeOMakinG — DISMISSED** [review]

**Comment by khouloudbelguith** [comment]
Hi @Hlavtox,

Can you please check the Conflicting files?

Thanks!

**Comment by Hlavtox** [comment]
@khouloudbelguith Rebased, QA please? :-)

**Review by NeOMakinG — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]
Thank you @Hlavtox

**Review by Progi1984 — DISMISSED** [review]

**Comment by Hlavtox** [comment]
@hibatallahAouadni rebased

**Review by jolelievre — DISMISSED** [review]

**Comment by Hlavtox** [comment]
@hibatallahAouadni @Progi1984 Can I rebase it after QA so I don't have to wait a week for approvals again? ;-)

**Comment by Progi1984** [comment]
@Hlavtox Sorry but no go for me (for this PR). Why ? Because changes in twig files can have an impact on your PR. So we need your rebase for checking QA with good elements.

Ping me when the rebase is done ;)

**Comment by Hlavtox** [comment]
@Progi1984 Rebased! ;-)

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by hibatallahAouadni** [comment]
Hello @Hlavtox 

Thanks for your patience and your PR :rocket: 
While testing your PR, I found this regression (even with PS1780 and develop) #26779

And I found this regression on your PR in Customer view page, but it seems it need to be fixed in the wishlist module, so I open a new issue for it #26780

Finally, LGTM :tada: QA :heavy_check_mark: 

Thanks!

**Comment by Progi1984** [comment]
Thanks @Hlavtox & @hibatallahAouadni

**Pull request merged**
